simulator/simulator.go

```diff
@@ -282,9 +281,9 @@ func getXcodeDeveloperDirPath() (string, error) {
 }
 
 // BootSimulator ...
-func BootSimulator(simulator InfoModel, xcodebuildVersion models.XcodebuildVersionModel) error {
+func BootSimulator(simulatorID string, xcodebuildMajorVersion int) error {
```

Comment:
This is a breaking change, will this be updated on all usage site now?

Reply:
and maybe here: https://github.com/bitrise-steplib/steps-install-certificate-to-simulator/blob/master/main.go#L105

Reply:
I'm happy to update the usage sites.